lms/templates/courseware/course_about_sidebar_header.html

```diff
@@ -9,44 +9,36 @@
 from six import text_type
 %>
 
-<header>
-  % if static.get_value('course_about_show_social_links', True):
-  <div class="social-sharing">
-    <div class="sharing-message">${_("Share with friends and family!")}</div>
-    ## TODO: this should probably be an overrideable block,
-    ##       or something allowing themes to do whatever they
-    ##       want here (and on this whole page, really).
-      <%
+    <header>
+        % if static.get_value('course_about_show_social_links', True):
+        <div class="social-sharing">
+            <div class="sharing-message">${_("Share with friends and family!")}</div>
+            ## TODO: this should probably be an overrideable block, ## or something allowing themes to do whatever they ## want here (and on this whole page, really).
+            <%
         site_domain = static.get_value('site_domain', settings.SITE_NAME)
         site_protocol = 'https' if settings.HTTPS == 'on' else 'http'
         platform_name = static.get_platform_name()
+        course_path = reverse('about_course', args=[text_type(course.id)])
+        course_url = f"{site_protocol}://{site_domain}{course_path}"
 
         ## Translators: This text will be automatically posted to the student's
         ## Twitter account. {url} should appear at the end of the text.
-        tweet_text = _("I just enrolled in {number} {title} through {account}: {url}").format(
+        tweet_text = _("I just enrolled in {number} {title} through {account} {url}").format(
             number=course.number,
             title=course.display_name_with_default,
             account=static.get_value('course_about_twitter_account', settings.PLATFORM_TWITTER_ACCOUNT),
-            url=u"{protocol}://{domain}{path}".format(
-                protocol=site_protocol,
-                domain=site_domain,
-                path=reverse('about_course', args=[text_type(course.id)])
-            )
+            url=course_url
         )
 
         tweet_action = u"https://twitter.com/intent/tweet?text={tweet_text}".format(tweet_text=six.moves.urllib.parse.quote(tweet_text))
 
-        facebook_link = static.get_value('course_about_facebook_link', settings.PLATFORM_FACEBOOK_ACCOUNT)
```

Comment:
where is this value passed in? can it be removed completely from there?

Reply:
It's passed as a href for for that <a> tag below

Reply:
sorry, confusing comment. what i meant was can we delete `course_about_facebook_link` from wherever it originates now? or is it used elsewhre?

Reply:
I don't think its defined somewhere... It tries to get it first from site configuration if its enabled, and if not or if its not defined in site configuration it will then use cascade `settings.PLATFORM_FACEBOOK_ACCOUNT`.

Reply:
It takes the get_value 
https://github.com/openedx/edx-platform/blob/master/common/djangoapps/pipeline_mako/templates/static_content.html#L14-L17
from static which import the get_value from site_configuration https://github.com/openedx/edx-platform/blob/677e0b8076d3e75def14fe25fe0c66c6a7788748/openedx/core/djangoapps/site_configuration/helpers.py#L119-L134